src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Color implements Display, Figure {
```

Comment:
Лучше, чтобы этот абстрактный класс назывался Figure. А Color - это будет enum с разными цветами (RED, GREEN, ...)
```suggestion
public abstract class Figure implements Drawable, AreaCalculator{
```